cmd/generate.go

```diff
@@ -0,0 +1,28 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/elastic/elastic-package/internal/cobraext"
+)
+
+const generateLongDescription = `Use this command to generate benchmarks data for a package. Currently, only data for what we have related assets on https://github.com/elastic/elastic-integration-corpus-generator-tool are supported.`
```

Comment:
maybe it is worth mentioning that this command has to be considered in beta: I'm not sure about the standard for beta/ga commands in elastic-package

@jsoriano any hint?

Reply:
As discussed, it is ok to add beta (or technical preview) commands. Please, add a mention to this here in the description, and consider logging an info message when this command is used.

Also, take into account that `elastic-package` is still 0.x, so we still reserve the right to make breaking changes to commands, although we try to avoid it.